src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/Binskim.linux-x64.dll.sarif

```diff
@@ -218,7 +218,28 @@
       },
       "invocations": [
         {
-          "executionSuccessful": true
+          "toolConfigurationNotifications": [
+            {
+              "locations": [
+                {
+                  "physicalLocation": {
+                    "artifactLocation": {
+                      "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Binskim.linux-x64.dll",
+                      "index": 0
+                    }
+                  }
+                }
+              ],
+              "message": {
+                "text": "'Binskim.linux-x64.dll' was not evaluated because its PDB could not be loaded (E_PDB_FORMAT)."
```

Comment:
>PDB could not be loaded [](http://example.com/codeflow?start=79&length=23)

In general, any transition you see from 'executionSuccessful' == true to the appearance of error notifications will be a usability problem. 

We need to chase down these 'PDB can't be loaded' issues. 

Several  high-level matters will be helpful to settle: 1) you need to make sure your PDB loading only occurs when the target is valid for your analysis, 2) you need to make sure your analysis only occurs if you observe the binary is built with a toolchain that provides source-link support.

i.e., we don't generally push for things like 'you need to update your stale compiler toolchain so you can enable sourcelink'. The pressure to upgrade is handled in the 'BuildWithSecureTools' rule. Assuming that rule is enforcing a min-version that also enables SourceLink, all we need to do is update that rule docs to mention SourceLink as yet another motivation to upgrade your compiler.

You may discover the min-version that rule enforces is too early for SourceLink support, in which case we should consider lifting it.